components/global/product_switcher_menu/product_switcher_menu.tsx

```diff
@@ -66,9 +66,10 @@ class ProductSwitcherMenu extends React.PureComponent<Props> {
 
         const {formatMessage} = this.props.intl;
 
+        // TODO: Ensure that clicking ItemLink menu items also closes the global header menu.
```

Comment:
Is this `TODO` needed? looks like something that should be already handled, right?

Reply:
Yep... I should remove it... thanks for pointing it out! 👊🏻